pkg/networkserver/utils.go

```diff
@@ -33,6 +34,46 @@ import (
 	"go.thethings.network/lorawan-stack/v3/pkg/unique"
 )
 
+var lorawanVersionPairs = map[ttnpb.MACVersion]map[ttnpb.PHYVersion]struct{}{
+	ttnpb.MAC_V1_0: {
+		ttnpb.PHY_V1_0: struct{}{},
+	},
+	ttnpb.MAC_V1_0_1: {
+		ttnpb.PHY_V1_0_1: struct{}{},
+	},
+	ttnpb.MAC_V1_0_2: {
+		ttnpb.PHY_V1_0_2_REV_A: struct{}{},
+		ttnpb.PHY_V1_0_2_REV_B: struct{}{},
+	},
+	ttnpb.MAC_V1_0_3: {
+		ttnpb.PHY_V1_0_3_REV_A: struct{}{},
+	},
+	ttnpb.MAC_V1_0_4: {
+		ttnpb.PHY_V1_0_3_REV_A: struct{}{},
```

Comment:
This one is not correct. There's a new PHY; see table https://github.com/TheThingsNetwork/lorawan-stack/issues/2784#issuecomment-648735723

Reply:
Which of these should be there then? https://github.com/TheThingsNetwork/lorawan-stack/blob/9566f042c64ee328a9026f85418fc393d50fe2dc/api/lorawan.proto#L82-L90
Or is it not supported yet?